src/SunPro.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SunPro"
+   name="SunPro License" listVersionAdded="3.20">
+      <crossRefs>
+         <crossRef>https://github.com/freebsd/freebsd-src/blob/main/lib/msun/src/e_acosh.c</crossRef>
+         <crossRef>https://github.com/freebsd/freebsd-src/blob/main/lib/msun/src/e_lgammal.c</crossRef>
+      </crossRefs>
+      <standardLicenseHeader>
+      </standardLicenseHeader>
+      <notes>
+      </notes>
+      <text>
+         <copyrightText>
+            Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
+         </copyrightText>
+         <p>
+            Developed at<alt name="SunOrg" match="SunPro|SunSoft">SunPro</alt>
+            , a Sun Microsystems, Inc. business.
```

Comment:
Two questions on these lines:
* should these be marked `<optional>`? I saw that the requested license text was just the "Permission to use..." bit below.
* might be worth adding a space between `at` and `<alt`? I'm not certain, but I suspect it _might_ render as those being right next to each other if a space is not included.

@jlovejoy I'm happy to add a commit to fix these, just wanted to ask before doing so!

Reply:
test/simpleTestForGenerator/SunPro.txt